src/Utils/html_utils.py

```diff
@@ -0,0 +1,53 @@
+import requests
+from bs4 import BeautifulSoup
+import requests
+from langchain.text_splitter import RecursiveCharacterTextSplitter, Language
+from bs4.element import Comment
+
+
+def read_html_file(path):
+    text_splitter = RecursiveCharacterTextSplitter(separators=["\n\n", "\n", ".", ";", ",", " ", ""],
```

Comment:
creation of RecursiveCharacterTextSplitter can be externalized. In fact, we should have some flexibility to adjust the chunk_size & chunk_overlap using the config file.